aiohttp/web_runner.py

```diff
@@ -218,3 +220,51 @@ def _unreg_site(self, site):
             raise RuntimeError("Site {} is not registered in runner {}"
                                .format(site, self))
         self._sites.remove(site)
+
+
+class ServerRunner(BaseRunner):
+    """Low-level web server runner"""
+
+    __slots__ = ('_web_server',)
+
+    def __init__(self, web_server, *, handle_signals=False, **kwargs):
+        super().__init__(handle_signals=handle_signals, **kwargs)
```

Comment:
Is there any need to duplicate `False` value even if base class has the same default?

Reply:
It is a tradeoff.
I believe duplication is better in terms of code readability and documentation.

Reply:
ok